config.go

```diff
@@ -41,3 +43,25 @@ type Configuration struct {
 	URL     string
 	Timeout time.Duration
 }
+
+// UnmarshalYAML parses a reporter configuration from YAML.
+func (configuration *Configuration) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
It's the idiomatic way of doing it ? I'm a bit surprised that we do not return a Configuration or an error ?

Reply:
Yes it it (cf https://godoc.org/gopkg.in/yaml.v2#Unmarshaler), it works by side effect (UnmarshalYAML receives a pointer to a struct and updates this struct).

Reply:
Thanks for the explanation 👍